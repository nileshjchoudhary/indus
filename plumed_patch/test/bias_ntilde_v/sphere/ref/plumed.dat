indus: INDUS INPUTFILE=indus.input TOL=100.0

# Restraint essenially serves as an intermediary to pass the forces 
# from ORDERPARAMETERS to the PLUMED core
restraint: RESTRAINT ARG=indus.ubias SLOPE=1.0 AT=0.0 KAPPA=0.0

PRINT ...
	LABEL=print
	ARG=indus.n,indus.ntilde,indus.ubias,restraint.bias,indus.maxnormderiv 
	STRIDE=1
	FILE=plumed.out
... PRINT
