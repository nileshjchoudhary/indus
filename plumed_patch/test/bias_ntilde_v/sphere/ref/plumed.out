#! FIELDS time indus.n indus.ntilde indus.ubias restraint.bias indus.maxnormderiv
 0.000000 132.000000 132.572883 8787.784706 8787.784706 0.000000
 1.000000 137.000000 137.504776 9453.781661 9453.781661 0.000000
 2.000000 137.000000 136.184506 9273.109828 9273.109828 0.000000
 3.000000 137.000000 138.490494 9589.808438 9589.808438 0.000000
 4.000000 137.000000 135.224234 9142.796729 9142.796729 0.000000
 5.000000 135.000000 133.666130 8933.317191 8933.317191 0.000000
 6.000000 132.000000 133.178801 8868.296459 8868.296459 0.000000
 7.000000 138.000000 137.655269 9474.486609 9474.486609 0.000000
 8.000000 136.000000 136.116336 9263.828420 9263.828420 0.000000
 9.000000 134.000000 133.758068 8945.610364 8945.610364 0.000000
 10.000000 135.000000 135.357360 9160.807427 9160.807427 0.000000
